pages/Appendixa.md

```diff
@@ -21,7 +21,7 @@ description: "Appendix A: Definitions"
 
 **Open Source License**: OSS is often associated with a license that details the terms and conditions governing the intellectual property rights of the software and its associated source code. These licenses specify how a particular work may be reproduced, modified, or used as a component of a larger system or as a standalone piece of software.<sup id="fnr49"><a href="#fn49">49</a></sup>
 
-**Open Source Software (OSS)**: Software that can be freely accessed, used, changed, and shared (in modified or unmodified form) by anyone. OSS is often distributed under licenses that comply with the definition of "Open Source" provided by the Open Source Initiative ([https://opensource.org/osd](https://opensource.org/osd)).<sup id="fnr50"><a href="#fn50">50</a></sup>
+**Free Software (OSS)**: Software that can be freely accessed, used, changed, and shared (in modified or unmodified form) by anyone. OSS is often distributed under licenses that comply with the definition of "Open Source" provided by the Open Source Initiative ([https://opensource.org/osd](https://opensource.org/osd)).<sup id="fnr50"><a href="#fn50">50</a></sup>
```

Comment:
New commit replaces OSS with FLOSS and related terms/definitions/references throughout